### PR TITLE
Add icons in menus

### PR DESCRIPTION
**Pull request by kRkk**
before:
before

after:
after

### CODE REVIEW

File: client/mainwindow.cpp [hunk 1]
```diff
@@ -207,24 +208,29 @@ void MainWindow::createMenu()
     roomSettingsAction->setDisabled(true);
     roomMenu->addSeparator();
     createRoomAction =
-        roomMenu->addAction(tr("Create &new room..."), [this]
+        roomMenu->addAction(QIcon::fromTheme("mail-message-new"),
+        tr("Create &new room..."), [this]
```
Comment (KitsuneRal) [inline client/mainwindow.cpp]: I'm not quite sure about this one, to be frank. An envelope really doesn't convey the right idea. I suppose `document-new` is the closest metaphor.

File: client/mainwindow.cpp [hunk 2]
```diff
@@ -207,24 +208,29 @@ void MainWindow::createMenu()
     roomSettingsAction->setDisabled(true);
```
Comment (KitsuneRal) [inline client/mainwindow.cpp]: While you're at it, adding an icon to `roomSettingsAction` (I propose `document-properties`) would be great.

File: client/mainwindow.cpp [hunk 3]
```diff
@@ -150,14 +150,15 @@ void MainWindow::createMenu()
     // Connection menu
     connectionMenu = menuBar()->addMenu(tr("&Accounts"));
 
-    connectionMenu->addAction(tr("&Login..."), this, [=]{ showLoginWindow(); } );
+    connectionMenu->addAction(QIcon::fromTheme("contact-new"), tr("&Login..."),
+        this, [=]{ showLoginWindow(); } );
```
Comment (KitsuneRal) [inline client/mainwindow.cpp]: `network-server` maybe? I know it's not the action icon, according to the spec, but using the same icon for logging in and opening a direct chat looks really confusing.

File: client/mainwindow.cpp [hunk 4]
```diff
@@ -305,7 +311,8 @@ void MainWindow::createMenu()
     );
 
     settingsMenu->addSeparator();
-    settingsMenu->addAction(tr("Configure &network proxy..."), [this]
+    settingsMenu->addAction(QIcon::fromTheme("network-wired"),
+        tr("Configure &network proxy..."), [this]
```
Comment (KitsuneRal) [inline client/mainwindow.cpp]: This should rather be `system-preferences-network`. Other network configuration options will be in the same dialog box as the network proxy configuration.

**Comment by kRkk** [comment]
Menu:
accounts menu
View menu:
view
Room menu:
room
Settings menu:
settings
Room list dock:
room list

**Comment by kRkk** [comment]
You can merge this

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Pull request merged**